app/docs/md/learn/concepts/images/component.md

```diff
@@ -0,0 +1,154 @@
+
+  The default value is `100vw`, which will propose that the browser choose an image size that will best fit the full width of the current viewport.
+
+  </dd>
+
+  <dt>width and height (optional)</dt>
+  <dd>
+
+  Each of these attributes takes a unitless length which should describe the intrinsic (not the rendered) width and height of your source image. The browser will use this to compute the aspect ratio of your image, which will help to avoid [Cumulative Layout Shift](https://web.dev/cls/). While these attributes are optional, we highly recommend filling them in, as this will improve performance and user experience.
+
+  </dd>
+</dl>
+
+## Examples
+
+### Basic usage
+
+```html
+<enhance-image
+  src='/_public/images/dog.jpg'
+  alt='My favourite dog'
+  width='3000'
+  height='2000'
+></enhance-image>
+```
+
+Using only the required attributes, and presuming no custom configuration has been specified in your `.arc` file, this will result in the following:
+
+- Transformations of your source image will be generated at 2400px, 1200px, and 800px wide
+- Each generated image will be formatted as webp, with an 80% quality setting
+- The browser will use whichever of these 3 generated images most closely fits the full width of the current viewport
+- The browser will use the provided width and height to determine the image's aspect ratio, thereby preventing cumulative layout shift
+
+### Providing `sizes`
+```html
+<enhance-image
+  src='/_public/images/dog.jpg'
+  alt='My favourite dog'
+  width='3000'
+  height='2000'
+  sizes='(min-width: 48em) 920px, 100vw'
+></enhance-image>
+```
+
+Again presuming the default configuration is being used in this example, the browser will select the generated image closest to the given width of 920px (in this case, the 1200px wide image) when the viewport is 48em or wider. At viewports narrower than 48em, the image closest to the full width of the current viewport will be used.
+
+### Custom configuration
+
+<doc-callout level="none" mark="🎛️">
+
+**[Read more about the customizing responsive images →](/docs/learn/concepts/images/customization)**
+
+</doc-callout>
+
+<doc-code filename="enhance.json">
+
+```json
+{
+  "plugins": {
+    "@enhance/image": {
+      "widths": [1280, 1024, 720, 480, 375],
+      "quality": 75,
+      "directory": "public/images/post-assets"
+    }
+  }
+}
+```
+
+</doc-code>
+
+```html
+<enhance-image
+  src='/_public/images/post-assets/dog.jpg'
+  alt='My favourite dog'
+  width='3000'
+  height='2000'
+  sizes='(min-width: 96em) 1200px, (min-width: 48em) 1000px, 100vw'
+></enhance-image>
+```
+
+- Bitmap images located in `public/images/post-assets` will be transformed ahead of runtime
```

Comment:
By `bitmap images` you mean binary images, not SVG's, right? This is for my own education.